TrafficCapture/trafficReplayer/src/main/java/org/opensearch/migrations/replay/TrafficReplayer.java

```diff
@@ -308,20 +310,19 @@ public static void main(String[] args) throws IOException, InterruptedException,
         }
 
         try (OutputStream outputStream = params.outputFilename == null ? System.out :
-                new FileOutputStream(params.outputFilename, true)) {
-            try (var bufferedOutputStream = new BufferedOutputStream(outputStream)) {
-                try (var blockingTrafficStream = TrafficCaptureSourceFactory.createTrafficCaptureSource(params,
-                        Duration.ofSeconds(params.lookaheadTimeSeconds))) {
-                    var tr = new TrafficReplayer(uri, buildAuthTransformerFactory(params),
-                            params.allowInsecureConnections, params.numClientThreads,  params.maxConcurrentRequests);
-                    setupShutdownHookForReplayer(tr);
-                    var tupleWriter = new SourceTargetCaptureTuple.TupleToFileWriter(bufferedOutputStream);
-                    var timeShifter = new TimeShifter(params.speedupFactor);
-                    tr.runReplayWithIOStreams(Duration.ofSeconds(params.observedPacketConnectionTimeout),
-                            blockingTrafficStream, bufferedOutputStream, timeShifter, tupleWriter);
-                    log.info("Done processing TrafficStreams");
-                }
-            }
+                new FileOutputStream(params.outputFilename, true);
+             var bufferedOutputStream = new BufferedOutputStream(outputStream);
+             var blockingTrafficStream = TrafficCaptureSourceFactory.createTrafficCaptureSource(params,
+                     Duration.ofSeconds(params.lookaheadTimeSeconds));
```

Comment:
Can we add another line here or something to distinguish that the try parenthesis has now ended

Reply:
You & I think alike ;).
I got fed up with too many resources and not knowing where the distinction was & did that in the last delinting PR.